app/src/components/TitleBar.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+    <nav class="navbar navbar-expand navbar-dark">
+        <a class='navbar-brand'>
+            <img v-bind:src="'https://www.aueb.gr/press/logos/1_AUEB-pantone-HR.jpg'"
+                 alt="logo" height="44" />
+        </a>
+        <nav class="navbar-nav mr-auto">
+            <img class="ml-2" v-bind:src="'https://www.dept.aueb.gr/schools_department_photos/cs.png'"
+                 alt="cs_logo" height="55" />
+            <a class="ml-4 navbar-brand mt-2">{{room}}</a>
+        </nav>
+        <a class="navbar-brand">
+            {{date}}
+        </a>
+    </nav>
+</template>
```

Comment:
This should be configurable!

Reply:
What?

Reply:
The header part should be fully configurable, to allow anyone who uses it to modify the colors, title and logos using the configuration.

Reply:
@root-expert

Reply:
Που και ποιο είναι το configuration?